src/p2/element.rs

```diff
@@ -1696,16 +1694,8 @@ pub fn input_from_properties(
         )?,
         placeholder: ftd::p2::utils::string_optional("placeholder", properties, doc.name, 0)?,
         multiline: ftd::p2::utils::bool("multiline", properties, doc.name, 0)?,
-        value: {
-            // if value is unchanged then
-            // value = default-value
-            // otherwise retain the changed value
-            match value {
-                Some(v) => Some(v),
-                None => default_value.clone(),
-            }
-        },
-        defaultvalue: default_value,
+        value: ftd::p2::utils::string_optional("value", properties, doc.name, 0)?,
+        defaultvalue: ftd::p2::utils::string_optional("default-value", properties, doc.name, 0)?,
```

Comment:
We use camelCase in Rust. `_` is also fine. Just merging two words is not ok.

Reply:
rust suggests to use snakecase like this default_value